assets/newsroom-management/App.tsx

```diff
@@ -91,7 +91,8 @@ class App extends React.Component<AppProps & DispatchProp<any>, AppState> {
           showWalletOnboarding={true}
           showWelcome={true}
           profileUrl={urls.PROFILE}
-          helpUrl={urls.HELP}
+          helpUrl={urls.FAQ_HOME}
+          baseUrl={urls.HELP_BASE}
```

Comment:
I'm going to add this comment to the `Civil` repo but just to point it out here, I think `baseUrl` is unclear as a variable name, maybe `helpUrlBase` instead.